src/dune_rules/odoc.ml

```diff
@@ -404,7 +413,7 @@ let setup_toplevel_index_rule sctx =
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
     <title>index</title>
-    <link rel="stylesheet" href="./odoc.css"/>
+    <link rel="stylesheet" href="./_odoc_support/odoc.css"/>
```

Comment:
How about creating a variable for this path and then substituting it here with sprintf?